### DISCUSSION TITLE
In All Times view, what is the default sort for tags?

### DISCUSSION THREAD

**Discussion by dhalinski-od** [kimai/kimai#5153] [Questions & Answers]
Could someone please tell me what is the default sort for tags in the All Times view? I expected it to be alpha, but it appears to be something else.

![20241107_Kimai_tag sort](https://github.com/user-attachments/assets/5767675b-4a46-4298-9b22-ed405575cb45)

**Comment by kevinpapst** [comment]
not sorted
